sadcowpng/sharedspaces/views.py

```diff
@@ -33,7 +29,7 @@ def client_sign_up(request):
             user = form.save(commit=False)
             user.is_client = True
             user.save()
-            return HttpResponseRedirect(reverse('/'))
+            return HttpResponseRedirect(reverse('index'))
```

Comment:
I am not fully sure but using index like this will leave the index in the URL so we might want to change it later. I am not too familiar with what the solution is but when I come across it again I will make sure to update this for you.

Reply:
I believe the reverse() uses the name from the urls.py to get its url, which worked for me but if there are any issues you can update at any time.